server/server/migrations/migrations/add_text_search_view_025.py

```diff
@@ -19,12 +19,6 @@ def create_view(self):
                             'lang': {'analyzers': ['identity']},
                         }
                     },
-                    'po_strings': {
```

Comment:
You shouldn't edit migrations ever, this has already executed on staging, you should create a new one that removes the field.